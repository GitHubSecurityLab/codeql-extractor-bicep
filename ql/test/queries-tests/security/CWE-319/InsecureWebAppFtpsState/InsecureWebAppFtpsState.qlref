security/CWE-319/InsecureWebAppFtpsState.ql
