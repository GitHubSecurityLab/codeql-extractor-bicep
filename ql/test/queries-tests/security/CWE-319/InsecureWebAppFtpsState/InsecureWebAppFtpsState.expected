| app.bicep:4:1:12:1 | AppService[webAppBad1] | Azure Web App allows insecure FTP protocol: AllAllowed. Use 'FtpsOnly' or 'Disabled' instead. |
| app.bicep:15:1:23:1 | AppService[webAppBad2] | Azure Web App allows insecure FTP protocol: FtpOnly. Use 'FtpsOnly' or 'Disabled' instead. |
